tests/certification/pubsub/azure/eventhubs/eventhubs_test.go

```diff
@@ -131,16 +131,16 @@ func TestEventhubs(t *testing.T) {
 			client := sidecar.GetClient(ctx, sidecarName)
 
 			// publish messages
-			ctx.Logf("Publishing messages. sidecarName: %s, topicName: %s", sidecarName, topicName)
+			logs := fmt.Sprintf("Published messages. sidecarName: %s, topicName: %s", sidecarName, topicName)
```

Comment:
This is still either present tense or future, cannot be past as the op is not complete yet.

Reply:
It's past as the log is printed after the messages have ben published only!